src/test/java/se/jsannemo/spooky/compiler/CompilerTest.java

```diff
@@ -77,4 +77,24 @@ void testCarl2() throws ParseException, ValidationException, InstructionExceptio
       ;
     Assertions.assertEquals("A", output.toString());
   }
+
+  @Test
+  void testGlobals() throws ParseException, ValidationException, InstructionException, VmException {
+    InputStream programStream =
+        getClass().getClassLoader().getResourceAsStream("example_programs/globals.spooky");
```

Comment:
I can't see this new program in the commit

Reply:
Woops!